strings/cas_white.txt

```diff
@@ -329,5 +329,20 @@ The robust economy system.
 Being roasted so devastatingly by the AI it counts as human harm.
 A turkey made of cybernetic hate.
 A thicc securitron.
+The gas giant.
+A Codex Cicatrix.
+The security department.
+A space dragon.
+Gender reassignment surgery.
```

Comment:
If anyone still has any objections pertaining to this line please post here.

Reply:
![image](https://user-images.githubusercontent.com/3204033/108345297-b2e6a500-71a3-11eb-85a4-b23dcd529368.png)

Reply:
Is this your objection?

Reply:
By your own admission you dismissed the above without addressing it to satisfaction.
![image](https://user-images.githubusercontent.com/3204033/108412413-7345aa80-71ef-11eb-8628-a09872f12b4d.png)

Reply:
So yeah, would rather not get wordlessly bonked into oblivion again by github over one line in an update to a two-year-old knock-off of CAH.

Our search function was BTFO by the ban, still hasn't recovered. Not worth the risk.

Reply:
I don't know why it was offensive but I replaced it with something that is better and has no chance of causing a ban that wouldn't happen without it being added.